mdx-models/src/main/java/com/mx/path/model/mdx/model/Resources.java

```diff
@@ -308,11 +308,11 @@ private static void registerProfileModelClasses(GsonBuilder builder) {
     // Update Password
     builder.registerTypeAdapter(Password.class, new ModelWrappableSerializer("password"));
     builder.registerTypeAdapter(new TypeToken<Password>() {
-    }.getType(), new ModelWrappableSerializer("update_password"));
+    }.getType(), new ModelWrappableSerializer("password"));
```

Comment:
We don't want to replace the old one, we need to add a new model and add a new entry in resources.

Reply:
So, create a entirely new object that will look exactly like the one we already have? Would that not be confusing?

Reply:
Also which name should I use? Since password is already taken.

Reply:
The problem isn't the class it is the serialization and deserialization. The root of the incoming and outgoing payload is defined by the class. I think the easiest way to vary it will be to create new, temporary classes for the new endpoints.

Reply:
Done.